### PR TITLE
[W4.4c][T15-A3]Alpheus Lim Wei Xiang

### PR DESCRIPTION
**Pull request by ALim95**

### CODE REVIEW

File: src/seedu/addressbook/storage/StorageFile.java [hunk 1]
```diff
@@ -9,14 +9,7 @@
...
 
-import java.io.BufferedReader;
-import java.io.BufferedWriter;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.Reader;
-import java.io.Writer;
+import java.io.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/storage/StorageFile.java]: don't import using `*` import only the classes necessary

File: src/seedu/addressbook/storage/StorageFile.java [hunk 2]
```diff
@@ -105,7 +98,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
             marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             marshaller.marshal(toSave, fileWriter);
 
-        } catch (IOException ioe) {
+        } catch (FileNotFoundException fnfe){
+            throw new InvalidStorageFilePathException("Read-only file");
```
Comment (okkhoy) [inline src/seedu/addressbook/storage/StorageFile.java]: you could create and use a custom exception instead of InvalidStorageFilePathException

**Comment by okkhoy** [comment]
@ALim95 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/storage/StorageFile.java (hunk 2 above)
Comment (ALim95) [reply src/seedu/addressbook/storage/StorageFile.java]: okay, so its better to just create a new exception called ReadOnlyFileException right? Got it!

File: src/seedu/addressbook/storage/StorageFile.java (hunk 1 above)
Comment (ALim95) [reply src/seedu/addressbook/storage/StorageFile.java]: okay got it!

**Pull request closed without merging**